waku/incentivization/rpc.nim

```diff
@@ -0,0 +1,17 @@
+import
+  json_serialization,
+  std/options
+import
+  ../waku_core
+
+# Exactly following the RFC:
+# https://github.com/vacp2p/rfc/tree/master/content/docs/rfcs/73
+
+type
+
+  EligibilityProof* = object
+    proof*: Option[seq[byte]]
```

Comment:
Not sure if this is a planned change in the RFC, but I'd stick to the same field names. In other words, this can `proofOfPayment` rather than just `proof`. If we add other proof types, these can be new fields.